python/dgl/graphbolt/impl/fused_csc_sampling_graph.py

```diff
@@ -934,6 +934,61 @@ def sample_negative_edges_uniform(
             ),
         )
 
+    def sample_negative_seeds_uniform(
+        self, edge_type, node_pairs, negative_ratio
+    ):
+        """
+        Sample negative edges by randomly choosing negative source-destination
+        edges according to a uniform distribution. For each edge ``(u, v)``,
+        it is supposed to generate `negative_ratio` pairs of negative edges
+        ``(u, v')``, where ``v'`` is chosen uniformly from all the nodes in
+        the graph. As ``u`` is exactly same as the corresponding positive edges,
+        it returns None for negative sources.
+
+        Parameters
+        ----------
+        edge_type: str
+            The type of edges in the provided node_pairs. Any negative edges
+            sampled will also have the same type. If set to None, it will be
+            considered as a homogeneous graph.
+        node_pairs : Tuple[Tensor, Tensor]
```

Comment:
Do we still prefer Tuple of Tensor, is it easier to pass the 2D tensor in directly?

Reply:
Now I pass the 2D tensor in directly, I will update docstring. I think we need a test later for determination.